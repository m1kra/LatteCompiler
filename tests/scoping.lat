
int main(){
    int x = 10, y;
    if (y > x){
        printString("Shouldn't print");
    } else {
        y = 12;
        int x = 30;
        printInt(x);
        printInt(y);
    }
    printInt(x);
    printInt(y);
    return 0;
}