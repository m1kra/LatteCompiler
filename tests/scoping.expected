30
12
10
12